setup.py

```diff
@@ -41,7 +41,8 @@
     install_requires = [
         'scipy',
         'pandas',
-        'matplotlib'
+        'matplotlib',
+        'tensorflow'
     ],
     license = 'NOSA',
```

Comment:
I assume you meant NASA not NOSA here?

Reply:
Actually it is supposed to be NOSA. NOSA stands for NASA Open Source Agreement